runner.sh

```diff
@@ -53,6 +75,17 @@ then
     echo
     echo "Installing Homebrew..."
     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+    # Homebrew on M1 macs installs to /opt/homebrew instead of /usr/local, so won't be picked up
+    # without some help
+    if "$IS_M1_MAC" && command -v brew 2 > /dev/null
```

Comment:
The redirect here isn't doing what's expected.

![image](https://user-images.githubusercontent.com/28069638/148963690-b7839e5b-c815-460b-a397-74e53bab722e.png)

Also, fun fact, [Posix defines the `-v` flag of `command`](https://man7.org/linux/man-pages/man1/command.1p.html) such that it doesn't write any output if there is no command found (it took me a while to get used to this because I had only used `type` in the past).


```suggestion
    if "$IS_M1_MAC" && command -v brew 2>/dev/null
```

Reply:
Ah, the `2` wasn't supposed to be there. My mistake.

Also, taking the space crunching idea in the latest commit.